### PR TITLE
Release v1.1.7

### PR DESCRIPTION
**Pull request by owenpearson**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+[Full Changelog](https://github.com/ably/ably-ruby/compare/v1.1.6...HEAD)
+
+**Implemented enhancements:**
+
+- Implement RSC7d \(Ably-Agent header\) [\#230](https://github.com/ably/ably-ruby/issues/230)
+- Support Ruby 3.x [\#220](https://github.com/ably/ably-ruby/issues/220)
+
+**Closed issues:**
+
+- Make CI Green [\#254](https://github.com/ably/ably-ruby/issues/254)
```
Comment (QuintinWillison) [inline CHANGELOG.md:14]: I would remove entries like this from issues and pull requests lists, which don't present any immediate customer-facing value. There is always the full changelog link if people are interested in every issue and pull request.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
 # Change Log
 
+## [v1.1.7](https://github.com/ably/ably-ruby/tree/v1.1.7)
+
+[Full Changelog](https://github.com/ably/ably-ruby/compare/v1.1.6...HEAD)
```
Comment (QuintinWillison) [inline CHANGELOG.md:5]: This line needs to be ready for the new tag after you land this to `main` branch.

```suggestion
[Full Changelog](https://github.com/ably/ably-ruby/compare/v1.1.6...v1.1.7)
```

**Review by QuintinWillison — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (AndyNicks) [reply CHANGELOG.md:14]: Agree with @QuintinWillison

Comment (owenpearson) [reply CHANGELOG.md]: done - d65eb00

**Review by QuintinWillison — APPROVED** [review]

**Review by AndyNicks — APPROVED** [review]

**Pull request merged**